### ISSUE TITLE
ImagePicker’s cropping does not work properly on iOS

### ISSUE THREAD

**Issue by lichenglu** [expo/expo#2061]
### Environment
```
Platform:
  iOS, non-detached, both in dev and standalone build.

Environment:
  OS: macOS Sierra 10.12.6
  Node: 8.9.1
  Yarn: 1.9.2
  npm: 6.1.0
  Watchman: Not Found
  Xcode: Xcode 9.2 Build version 9C40b
  Android Studio: Not Found

Packages: (wanted => installed)
  expo: ^29.0.0 => 29.0.0
  react: 16.3.1 => 16.3.1
  react-native: https://github.com/expo/react-native/archive/sdk-29.0.0.tar.gz => 0.55.4
```
Although the environment lists `expo 29`, but I am using `expo 28` for reproduction, as currently there seems to be an issue of using media library in dev for sdk 29. But I can confirm the standalone that uses sdk 29 also has this issue

### Steps to Reproduce

1. Open `snack` [here](https://snack.expo.io/@lichenglu/cropping-reproduction). Choose `iOS` for running.
2. Tap `Open camero roll` text after render finishes.
3. Grant permission for cameraroll
4. Wait until cameraroll shows up, then select the image named `Djupavogshreppur` under `Moments` category (view of waterfall). This is important, as the dimension of image is big enough.
5. Enter the `editing (cropping) screen`. Try to scroll the `crop box` to the top most of the image
6. Crop box fails to reach the top edge of the image.

### Expected Behavior

Should be able to crop the top edge of an image.

It might be the pivot of the cropping box is off? Or something related to the scrollview?

### Actual Behavior

Crop box failed to reach the top edge of the image.

[A gif demo](https://drive.google.com/file/u/1/d/1hXhZQRwuk51uM_X82o7Fbc8QhHTYfany/view?usp=drive_web). It could take a while to load, quite big.

### Reproducible Demo

[Snack](https://snack.expo.io/@lichenglu/cropping-reproduction)

**Comment by lichenglu** [comment]
OK, just check the image picker module of expo. It seems that expo is using iOS' `UIImagePickerController` directly. After researching a bit, the issue I described seems to be a "bug" of iOS since [iOS 6.0](https://stackoverflow.com/questions/12630155/uiimagepicker-allowsediting-stuck-in-center). Apple, really?

The only solution might be to implement a custom cropping view controller. Any plan on doing it? Or I can do it, though I am more familiar with Swift than ObjC.

**Comment by ccheever** [comment]
Yeah, it is strange that they have left it that way.

You can use the `Camera` module in Expo and combine it with the Expo `ImageManipulator` and write something yourself that behaves the way you want it to.

https://docs.expo.io/versions/latest/sdk/imagemanipulator#__next

Someone may have already done this, if you Google around.

Will close this for now since this just exposes the system image picker and we are waiting on a fix from them.
